docs/datadog_monitor.md

```diff
@@ -51,6 +52,18 @@ To deploy a `DatadogMonitor` with the Datadog Operator, use the [`datadog-operat
     ```
 
     For additional examples, see [examples/datadog-monitor](../examples/datadogmonitor). Note that only metric alerts, query alerts, and service checks are supported.
+   - Note: By default the Operator only watches its own `namespace`, so it will manage any `DatadogAgent` and `DatadogMonitor` objects within its own `namespace`. So you should deploy your Datadog objects in the same `namespace` as the Operator.
```

Comment:
```suggestion
   - Note: By default, the Operator only watches its own `namespace`, so it will manage any `DatadogAgent` and `DatadogMonitor` objects within its own `namespace`. Therefore, you should deploy your Datadog objects in the same `namespace` as the Operator.
```